src/main/java/com/ksm/wstbackoffice/enums/PriceType.java

```diff
@@ -0,0 +1,12 @@
+package com.ksm.wstbackoffice.enums;
```

Comment:
try not to use package names in the plural.

Reply:
done